recipes/impedancefitter/meta.yaml

```
@@ -0,0 +1,53 @@
+{% set name = "impedancefitter" %}
+{% set version = "2.0.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/impedancefitter-{{ version }}.tar.gz
+  sha256: 30eac8eb8d7230cc948eefdfad638239f09af0efb40d96a6e0e8c758811e6279
+
+build:
+  skip: true
```

```suggestion
  noarch: python
```
Currently, everything is skipped.